plotUtils/compareSN.C

```diff
@@ -0,0 +1,73 @@
+#include "TH1.h"
+#include "TH2.h"
+#include "TCanvas.h"
+#include "TFile.h"
+#include <iostream>
+#include <cmath>
+#include <cstdlib>
+using namespace std;
+
+
+string sampleDir="/home/fynu/obondu/Higgs/CommonTools/plotUtils";
+string TTFile = sampleDir+"/TTJets_TuneCUETP8M1_amcatnloFXFX_25ns_15c5fa1_c106444_histos.root";
+string DYFile = sampleDir+"/DYJetsToLL_M-50_TuneCUETP8M1_13TeV-amcatnloFXFX_Asympt25ns_15c5fa1_c106444_histos.root";
+
+double L_exp = 1. ; //Expected lumi
+
+double X_signal = 1. ; //Xsection in fb
```

Comment:
it is safer to put everything in pb, this is the basic unit in every MC sample we produce...